dev-client/src/screens/CreateSiteScreen/components/CreateSiteView.tsx

```diff
@@ -68,7 +68,7 @@ export const CreateSiteView = ({
       const createdSite = await createSiteCallback({...site, elevation});
       if (createdSite !== undefined) {
         homeScreen?.showSiteOnMap(createdSite);
-        navigation.pop();
+        navigation.navigate('BOTTOM_TABS');
```

Comment:
Can you explain the difference here between these two behaviors? / why this was necessary for this PR

Reply:
When you create after tapping Learn More, you need to go back more than one screen. If you create a site by tapping Create Site directly, you only need to go back one screen. In both cases, you want to go back to the home screen.